.gitignore

```diff
@@ -3,6 +3,7 @@
 .AppleDouble
 .LSOverride
 node_modules
+nodejs
```

Comment:
won't this break the layer? why did we remove all the files?

Reply:
Ah your right we'll need to commit. I was thinking if it built locally it would reference the node modules but that wont be the case in the repo. Ill add it back

Reply:
Note that what I had in the previous commit is what it needs to be - I built that using the correct versions based off of the npm package instructions

Reply:
Isn't the layer built on the build phase, `npm install && npm run build`?